test/test_files/copy/copy_edge_case.test

```diff
@@ -0,0 +1,22 @@
+-GROUP CopyEdgeCaseTest
```

Comment:
I don't think this file/group name makes sense. These aren't edge cases. These are long value tests.

Reply:
We are testing when the strings and lists are long, i think this is an edge case.